pkg/syncer/volume_health.go

```diff
@@ -111,7 +111,7 @@ func csiGetVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 									newPvc.Namespace, newPvc.Name, val, volHealthStatus)
 								metav1.SetMetaDataAnnotation(&newPvc.ObjectMeta, annVolumeHealth, volHealthStatus)
 								metav1.SetMetaDataAnnotation(&newPvc.ObjectMeta, annVolumeHealthTS, time.Now().Format(time.UnixDate))
-								log.Infof("set annotation for health to %s at time %s", volHealthStatus, time.Now().Format(time.UnixDate))
+								log.Infof("csiGetVolumeHealthStatus: set annotation for health to %s at time %s for pvc %s/%s ", volHealthStatus, time.Now().Format(time.UnixDate), newPvc.Namespace, newPvc.Name)
```

Comment:
Edit the log on line 100 as well to keep it consistent. Also, we already have a debug log capturing the same info(on line 109) . So do we even need this log?

Reply:
We need this to debug in production. I recently triaged a bug where there was no log about which PVC the annotation is set for

Reply:
Then should we remove the debug log? It's redundant.

Reply:
Debugf is logging more details like old value and current value of health status, and also details like get() failed on the api server. I think it's ok to have Debug log. 
Let me know if you still think we need to remove it.

Reply:
Ok sure.

Reply:
Can we just convert DEBUG log to INFO log? It provides more information which is useful to triage.

Reply:
I have made debug log at line:109 as info and removed line:114 as it was redundant